packages/client/src/pages/SignIn/index.js

```diff
@@ -19,7 +19,7 @@ const SignInPage = () => {
 
     return (
         <>
-        { JSON.stringify(user) }
+        { JSON.stringify(user, null, 4) }
```

Comment:
Code cleanness - Avoid leaving debugging data, we can just assume the user object is available, and render it in the appropriate component if needed.

Reply:
Create this page for debugging. Probably would delete it before merging the PR since Ojasw is working on the sign in component.

Reply:
When I made the Sign-in page, I think I deleted it. I will probably create a PR by the end of today to merge the component over.